### ISSUE TITLE
Add election notice field(s) to the Election object

### ISSUE THREAD

**Issue by afsmythe** [votinginfoproject/vip-specification#401]
States participating in VIP are requesting that some custom language be surfaced to notify users of last minute changes to voting options (including increased absentee voting). We are presently suggesting that the `VoterService` fields be used for this use case, and it's become evident that this type of information might be better suited to be tied to the `Election` object. Election office's responses to COVID-19 is the most obvious use case, but we envision such a field to also to be used for natural disasters related to weather. 

The proposal is to add a complex type `ElectionNotice` to the `Election` object with the following elements:
- NoticeText of type `InternationalizedText`
- NoticeURI of type `AnyURI`

Possible implementations of this would be a banner or pop up message on websites consuming VIP data to relay official information from the States or Counties for an Election.

**Comment by afsmythe** [comment]
Gentle ping on this for input. Below is what an `Election` XML might look like to support this using a `ElectionNotice` complex type:
```
<Election>
   <ElectionNotice>
      <NoticeText>
          <Text language="en">blah blah</Text>
          <Text language="es">blah blah</Text>
      </NoticeText>
      <NoticeURI>https://www.blah.gov</NoticeURI>
   </ElectionNotice>
   <..>
</Election>
```
Or I think we'd also be in good shape if these fields were simply made attributes of the `Election` object:
```
<Election>
   <ElectionNoticeText>
      <Text language="en">blah blah</Text>
      <Text language="es">blah blah</Text>
   </ElectionNoticeText>
   <ElectionNoticeURI>https://www.blah.gov</ElectionNoticeURI>
   <..>
</Election>
```
Interested to hear any feedback!

**Comment by afsmythe** [comment]
Agree on point 1). 

For 2) this would open up the possibility of there being multiple notices within a single VIP feed, since multiple `ElectionAdministration` records are allowed. Only one `Election` is allowed in a feed, and so assigning the notice to `Election` would ensure there is only one notice. 

I suppose the concern around 2) could be mitigated by an implementation of the API where the notice aligned with state[].electionAdministrationBody would be surfaced instead of any notice in state[].local_jurisdiction. Thoughts?

**Comment by jdmgoogle** [comment]
Are there ever situations where a state would want to add an emergency/urgent notification, and the county would have additional/other county-specific information as well? Or is the intention that the state would be responsible for conveying any and all county-specific information in their top-level notification?

**Comment by JDziurlaj** [comment]
From a purely semantic perspective it seems like this would make the most sense under `ElectionAdministration`. There could be notices at the locality level, perhaps not so much for COVID, but natural disasters and other situations that are more localized. I like leaving the name `ElectionNotice` as-is, and convey your intent through documentation.

**Comment by afsmythe** [comment]
There's a valid point about localized notices. Let's tie the notice fields to `ElectionAdministration` with the intent that the State and a Locality are able to provide notice and that each of these should be surfaced if they are both present. With sufficient QA review we should be able to identify conflicting information for the VIP data provider to correct. 

Looks like we just need agreement on the field names. Is the 1500-100 concern something that needs to be addressed for this 5.2 revision?

**Comment by JDziurlaj** [comment]
No. 1500-100 v2 is published and I don't expect another rev (save for bugfixes) until 2021 at the earliest.
